index.html

```diff
@@ -7,92 +7,226 @@
+            </li>
+            <li class="numbers_fourth_page">
+              <a class="anchor_number" href="#">2</a>
+            </li>
+            <li class="numbers_fourth_page">
+              <a class="anchor_number" href="#">3</a>
+            </li>
+          </ul>
+        </div>
       </div>
-      <div class="btn"><button>Book now</button></div>
-    </div>
-    <div class="right_side">
-      <img class="mountains_pic" src="mountain_home.png" alt="widok na góry">
-    </div>
-  </div>
-  <div class="fourth_page_container">
-    <div class="left_side_fourth_page">
-      <div><img class="girl_pic" src="girl.png" alt="zdjęcie dziewczyny"></div>
+      </div>
+    </section>
+
+
+
+    <section class="section fifth_page_container">
+      <div class="flex section_wrapper">
       <div class="newsletter">
-        <label for="newsletter_header"> newsletter comming soon</label>
-        <input type="email" class="email" placeholder="your e-mail address">
+        <div class="input_newsletter">
+          <div class="input_header">
+            <label for="newsletter_header" class="newsletter_info"> newsletter comming soon</label>
+          </div>
+          <div class="input_input">
+            <input type="email" class="email" placeholder="your e-mail address">
+          </div>
+        </div>
       </div>
-    </div>
-    <div class="right_side_fourth_page">
-      <h1 class="cabin_lovers">+2,000<br>cabin lovers</h1>
-      <p class="fourth_page_footer">Marina Skovic, Visual Artist</p>
       <div class="bot_nav_container">
-        <nav class="bot_nav"></nav>
+        <ul class="ul_container">
+          <li class="nav"><a class="bot_list" href="#">Help Center</a></li>
+          <li class="nav"><a class="bot_list" href="#">Facebook</a></li>
+          <li class="nav"><a class="bot_list" href="#">About</a></li>
+        </ul>
+        <ul class="ul_container">
+          <li class="nav"><a class="bot_list" href="#">FAQ</a></li>
+          <li class="nav"><a class="bot_list"  href="#">Contact</a></li>
+          <li class="nav"><a class="bot_list" href="#">Instagram</a></li>
+        </ul>
+        <ul class="ul_container">
+          <li class="nav"><a class="bot_list" href="#">Blog</a></li>
+          <li class="nav"><a class="bot_list" href="#">Reviews</a></li>
+          <li class="nav"><a class="bot_list" href="#">Twitter</a></li>
       </div>
     </div>
-  </div>
+    </section>
 
-</body>
 
 
+    <footer class="section section_wrapper">
+      <div class="last_footer_wrapper">
+        <div class="last_footer">
+          <div class="footer_box">
+            <ul class="footer_list">
+              <li class="footer_par"><a class="foot_link" href="#">&copy; 2020 Remotely</a></li>
+              <li class="footer_par"><a class="foot_link" href="#">Privacy Policy</a></li>
+              <li class="footer_par"><a class="foot_link" href="#">Terms and Conditions</a></li>
+            </ul>
```

Comment:
24px of free space between the page end and those links on Figma. Pls check this

Reply:
fixed